packages/ses/README.md

```diff
@@ -27,34 +27,79 @@ npm install ses
 
 ## Usage
 
-### Module
+### Lockdown
 
-This example locks down the current realm, turning it into a starting
-compartment.
-Within a compartment, there is a `Compartment` constructor that conveys
-"endownments" into the new compartment's global scope, and a `harden` method
-that that object and any object reachable from its surface.
-The compartment can import modules and evaluate programs.
+SES introduces the `lockdown` function.
+Alters the surrounding execution environment such that no object intrinsically
```

Comment:
grammar/readability nits: maybe:

* Calling `lockdown` alters the surrounding..
* .. no intrinsically-accessible object, like ..
* . `lockdown` also tames other ..